pkg/apis/extensions/types.go

```diff
@@ -296,6 +296,12 @@ type DeploymentStatus struct {
 
 	// Total number of non-terminated pods targeted by this deployment that have the desired template spec.
 	UpdatedReplicas int `json:"updatedReplicas,omitempty"`
+
+	// Total number of pods targeted by this deployment that are ready.
+	ReadyPods int `json:"readyPods,omitempty"`
```

Comment:
If this doesn't match the number of pods with the Ready condition True, then we should call it AvailablePods (and UnavailablePods), to match MaxUnavailable.